dlrover/python/unified/util/os_util.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2025 The DLRover Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from dlrover.python.common.log import default_logger as logger
+from dlrover.python.util.common_util import (
+    find_free_port_in_range,
+    find_free_port_in_set,
+)
+
+
+def get_free_port() -> int:
```

Comment:
already defined in base util package

Reply:
This is copied from `training`, and it's not same as `base util` ones.